src/main/java/org/terasology/launcher/gui/javafx/FXUtils.java

```diff
@@ -40,4 +42,65 @@ static ScaleTransition createScaleTransition(final double factor, final Node nod
         return scaleTransition;
     }
 
+    /**
+     * Provides factory methods for timers that are manipulated from and execute their action on the JavaFX application
+     * thread.
+     * <p>
+     * Derived from ReactFX.
+     *
+     * @see <a href="https://github.com/TomasMikula/ReactFX/blob/v2.0-M5/reactfx/src/main/java/org/reactfx/util/FxTimer.java">ReactFX</a>
+     */
+    public static final class FxTimer {
+        private final Duration actionTime;
+        private final Timeline timeline;
+        private final Runnable action;
+
+        private long seq;
```

Comment:
Looks to be some kind of sequential identifier?

(This is code straight from ReactFX, so I know less about it even compared to my own code I've written and since forgotten the details of.)